src/Microsoft.OpenApi/Reader/V3/OpenApiSchemaDeserializer.cs

```diff
@@ -52,13 +52,13 @@ internal static partial class OpenApiV3Deserializer
                 (o, n, _) => o.IsExclusiveMaximum = bool.Parse(n.GetScalarValue())
             },
             {
-                "minimum",
+               "minimum",
                 (o, n, _) =>
                 {
                     var min = n.GetScalarValue();
-                    if (min != null)
+                    if (!string.IsNullOrEmpty(min))
                     {
-                        o.Minimum = ParserHelper.ParseDecimalWithFallbackOnOverflow(min, decimal.MinValue);
```

Comment:
out of curiosity, do we still need the helper method? can we remove it?

Reply:
No we don't need it anymore. Resolving...